webui/tests/e2e-cluster/Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: clean
+clean:
+	rm -rf build/
+
+.PHONY: start-db
+start-db: stop-db
+	# TODO make the db port configurable
```

Comment:
name it test-cluster and remove TODOs ?

Reply:
maybe `start-test-cluster` and `stop-test-cluster`?

removing the TODO sounds good.

Reply:
by `test-cluster` here I mean the directory name. I rather not change the target names: keep it a direct copy from make tools

Reply:
agreed, didn't realize back then this was a copy of the `tools` target.